graphql/util.go

```diff
@@ -532,16 +532,16 @@ func generateBuildVariants(sc data.Connector, versionId string, searchVariants [
 	timeToSortTasks := time.Since(sortTasksStartTime)
 
 	totalTime := time.Since(start)
-	grip.InfoWhen(totalTime > time.Second*2, message.Fields{
-		"Ticket":            "EVG-14828",
-		"timeToFindTasks":   timeToFindTasks,
-		"timeToBuildTasks":  timeToBuildTasks,
-		"timeToGroupTasks":  timeToGroupTasks,
-		"timeToSortTasks":   timeToSortTasks,
-		"totalTime":         totalTime,
-		"versionId":         versionId,
-		"taskCount":         len(tasks),
-		"buildVariantCount": len(result),
+	grip.InfoWhen(time.Duration(totalTime.Milliseconds()) > time.Second*2, message.Fields{
+		"Ticket":             "EVG-14828",
+		"timeToFindTasksMS":  timeToFindTasks.Milliseconds(),
```

Comment:
Wont this still be time.duration just converted to Milliseconds?

Reply:
(instead of nanoseconds)

Reply:
https://go.dev/play/p/F9dtew60dob

Reply:
I think we use milliseconds in some other splunk logging it's also arguably a more readable value.

Reply:
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/4605522/154376591-6d4f3f44-88ef-4d96-ad21-9ce44df21032.png">